WORKSPACE

```diff
@@ -246,3 +246,29 @@ http_archive(
 load("@hedron_compile_commands//:workspace_setup.bzl", "hedron_compile_commands_setup")
 
 hedron_compile_commands_setup()
+
+#Antlr
+
+http_archive(
+    name = "rules_antlr",
+    sha256 = "26e6a83c665cf6c1093b628b3a749071322f0f70305d12ede30909695ed85591",
+    strip_prefix = "rules_antlr-0.5.0",
+    urls = ["https://github.com/marcohu/rules_antlr/archive/0.5.0.tar.gz"],
+)
+
+load("@rules_antlr//antlr:repositories.bzl", "rules_antlr_dependencies")
+load("@rules_antlr//antlr:lang.bzl", "CPP")
+
+rules_antlr_dependencies("4.8")
+
+ANTLR4_RUNTIME_GIT_SHA = "70b2edcf98eb612a92d3dbaedb2ce0b69533b0cb"
+
+ANTLR4_RUNTIME_SHA = "fae73909f95e1320701e29ac03bab9233293fb5b90d3ce857279f1b46b614c83"
+
+http_archive(
+    name = "antlr4_runtimes",
+    build_file = "@//third_party/antlr:antlr.BUILD",
+    sha256 = ANTLR4_RUNTIME_SHA,
+    strip_prefix = "antlr4-" + ANTLR4_RUNTIME_GIT_SHA,
+    urls = ["https://github.com/antlr/antlr4/archive/" + ANTLR4_RUNTIME_GIT_SHA + ".tar.gz"],
+)
```

Comment:
It looks like the `rules_antlr_dependencies` is quoting a particular released version of Antlr while we are using an arbitrary SHA to get the Antlr project.

Since you are using an official release version number in the `rules_antlr_dependencies` invocation, I think it makes sense to also grab a tarball associated with an official release. I'd say update this to grab a file at a URL associated with a release number, as that will make it easier to keep the dependencies and the code in sync.

Reply:
Actually, do you even need to pull this in separately? It looks like `rules_antlr` pulls this on its own:https://github.com/marcohu/rules_antlr/blob/89a29cca479363a5aee53e203719510bdc6be6ff/antlr/repositories.bzl#L40

Reply:
antlr4_runtime provided in the github repo mentioned above do not have a feature exclusively enabling exceptions in the build, copts = ["-fexceptions"]. Hence, used our own.

Reply:
Updated to latest Antlr runtime version of 4.9.3. I was seeing couple of -Werrors on 4.8 version. The SHA version we were fetching was different from the 4.8.tar.gz which didn't make sense to me and there was no reason for us to not move to the latest Antlr version. So instead of trying to debug and fix the -Werrors, I tried moving to 4.9.3 which worked out perfectly for us. rules_antlr however still has dependency on 4.8 and they don't support 4.9 yet.